app/helpers.php

```diff
@@ -427,6 +427,34 @@ function get_class_namespace($className)
     return substr($className, 0, strrpos($className, '\\'));
 }
 
+function ci_file_search($fileName)
+{
+    if (file_exists($fileName)) {
+        return $fileName;
+    }
+
+    $directoryName = dirname($fileName);
+    $fileArray = glob($directoryName.'/*', GLOB_NOSORT);
+    $fileNameLowerCase = strtolower($fileName);
+    foreach ($fileArray as $file) {
+        if (strtolower($file) === $fileNameLowerCase) {
+            return $file;
+        }
+    }
+
+    return false;
+}
+
+function deltree($dir)
+{
+    $files = array_diff(scandir($dir), ['.', '..']);
+    foreach ($files as $file) {
+        (is_dir("$dir/$file")) ? delTree("$dir/$file") : unlink("$dir/$file");
```

Comment:
Oh look the function name is wrong. Except it works anyway because method name is case insensitive :dancer: